diff --git a/lib/typegen.js b/lib/typegen.js
index d82821491cabd566885ee6e41f4fbfdc1276c315..a7cb00eea7522d13cabb6c866f90b773b050834b 100644
--- a/lib/typegen.js
+++ b/lib/typegen.js
@@ -71,7 +71,7 @@ class Typegen {
             for (let [name, versions] of (0, util_1.groupBy)(palletItems, it => it.def.name)) {
                 out.line();
                 out.block(`export const ${(0, util_1.toJsName)(name)} = `, () => {
-                    out.line(`name: '${pallet}.${name}',`);
+                    out.line(`name: '${pallet}.${name}' as const,`);
                     for (let it of versions) {
                         let useSts = file.useSts(it.runtime);
                         out.blockComment(it.def.docs);
