diff --git a/node_modules/@subsquid/substrate-typegen/lib/typegen.js b/node_modules/@subsquid/substrate-typegen/lib/typegen.js
index 631781c..6a5d23f 100644
--- a/node_modules/@subsquid/substrate-typegen/lib/typegen.js
+++ b/node_modules/@subsquid/substrate-typegen/lib/typegen.js
@@ -71,7 +71,7 @@ class Typegen {
             for (let [name, versions] of (0, util_1.groupBy)(palletItems, it => it.def.name)) {
                 out.line();
                 out.block(`export const ${(0, util_1.toJsName)(name)} = `, () => {
-                    out.line(`name: '${pallet}.${name}',`);
+                    out.line(`name: '${pallet}.${name}' as const,`);
                     for (let it of versions) {
                         let useSts = file.useSts(it.runtime);
                         out.blockComment(it.def.docs);
