diff --git a/node_modules/prom-client/lib/metrics/eventLoopLag.js b/node_modules/prom-client/lib/metrics/eventLoopLag.js
index 45d5c33..aa8a847 100644
--- a/node_modules/prom-client/lib/metrics/eventLoopLag.js
+++ b/node_modules/prom-client/lib/metrics/eventLoopLag.js
@@ -6,7 +6,7 @@ const Gauge = require('../gauge');
 let perf_hooks;
 try {
 	/* eslint-disable node/no-unsupported-features/node-builtins */
-	perf_hooks = require('perf_hooks');
+	// perf_hooks = require('perf_hooks');
 } catch {
 	// node version is too old
 }
