diff --git a/node_modules/typeorm/query-builder/SelectQueryBuilder.js b/node_modules/typeorm/query-builder/SelectQueryBuilder.js
index 93c877e..fc7ffcf 100644
--- a/node_modules/typeorm/query-builder/SelectQueryBuilder.js
+++ b/node_modules/typeorm/query-builder/SelectQueryBuilder.js
@@ -2449,7 +2449,7 @@ class SelectQueryBuilder extends QueryBuilder_1.QueryBuilder {
                         const existJoin = this.joins.find((join) => join.alias === joinAlias);
                         if (!existJoin) {
                             this.joins.push({
-                                type: "inner",
+                                type: "left",
                                 select: false,
                                 selection: undefined,
                                 alias: joinAlias,
@@ -2457,10 +2457,6 @@ class SelectQueryBuilder extends QueryBuilder_1.QueryBuilder {
                                 relationMetadata: relation,
                             });
                         }
-                        else {
-                            if (existJoin.type === "left")
-                                existJoin.type = "inner";
-                        }
                         const condition = this.buildWhere(where[key], relation.inverseEntityMetadata, joinAlias);
                         if (condition) {
                             andConditions.push(condition);
